src/components/Lightboxes/AddToBasketInterstitial.vue

```diff
@@ -93,50 +96,70 @@ import cookieStore from '@/util/cookieStore';
 import basketAddInterstitial from '@/graphql/query/basketAddInterstitialClient.graphql';
 import basketAddInterstitialData from '@/graphql/query/basketAddInterstitialData.graphql';
 import updateAddToBasketInterstitial from '@/graphql/mutation/updateAddToBasketInterstitial.graphql';
-import KvLightbox from '@/components/Kv/KvLightbox';
 import KvButton from '@/components/Kv/KvButton';
+import KvCheckbox from '@/components/Kv/KvCheckbox';
+import KvLightbox from '@/components/Kv/KvLightbox';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
 import LoanReservation from '@/components/Checkout/LoanReservation';
 import LYML from '@/components/LoansYouMightLike/lymlContainer';
-import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
 
 export default {
 	components: {
-		KvLightbox,
 		KvButton,
+		KvCheckbox,
+		KvLightbox,
+		KvLoadingSpinner,
 		LoanReservation,
 		LYML,
-		KvLoadingSpinner,
 	},
 	inject: ['apollo'],
 	data() {
 		return {
-			showInterstitial: false,
 			basketInterstitialState: {},
 			loan: {},
 			loans: () => [],
 			loanCount: 0,
 			loanTotals: '0.00',
 			loading: true,
+			showInterstitial: false,
+			userPrefHideInterstitial: false,
 		};
 	},
 	computed: {
 		basketInterstitialActive() {
 			return this.basketInterstitialState.active || false;
 		}
 	},
+	watch: {
+		userPrefHideInterstitial() {
+			try {
+				localStorage.setItem('userPrefHideInterstitial', this.userPrefHideInterstitial);
+
+				console.log(`TRACK: Lending, click-hide-add-to-basket-interstitial, ${this.userPrefHideInterstitial ? 'selected' : 'unselected'}`); // eslint-disable-line max-len
+				this.$kvTrackEvent(
+					'Lending',
+					'click-hide-add-to-basket-interstitial',
+					this.userPrefHideInterstitial ? 'selected' : 'unselected'
+				);
+			} catch (err) {
+				console.error(err);
+			}
+		}
+	},
 	mounted() {
 		this.apollo.watchQuery({ query: basketAddInterstitial }).subscribe({
 			next: ({ data }) => {
 				const interstitialState = _get(data, 'basketAddInterstitial');
-				this.showInterstitial = interstitialState.active ? interstitialState.visible : false;
-
 				this.basketInterstitialState = {
 					...this.basketInterstitialState,
 					active: interstitialState.active,
 					visible: interstitialState.visible,
 					loanId: interstitialState.loanId,
 				};
 
+				this.userPrefHideInterstitial = localStorage.getItem('userPrefHideInterstitial') === 'true';
```

Comment:
Because of the `watch` on this property, setting this here will cause the tracking event to fire. I would recommend tracking/setting the value in storage using a click handler, rather than a watch.